packages/ujson/meta.yaml

```diff
@@ -0,0 +1,129 @@
+package:
+  name: ujson
+  version: 5.10.0
+  top-level:
+    - ujson
+source:
+  url: https://files.pythonhosted.org/packages/f0/00/3110fd566786bfa542adb7932d62035e0c0ef662a8ff6544b6643b3d6fd7/ujson-5.10.0.tar.gz
+  sha256: b3cd8f3c5d8c7738257f1018880444f7b7d9b66232c64649f562d7ba86ad4bc1
+about:
+  home: https://github.com/ultrajson/ultrajson
+  PyPI: https://pypi.org/project/ujson
+  summary: Ultra fast JSON encoder and decoder for Python
+  license: "Developed by ESN, an Electronic Arts Inc. studio.
```

Comment:
Could you summarize the license with a shorter sentence? I know that this is auto-generated field, but it is a bit too long. The license field here doesn't have to be a full license.

Reply:
Done based on @agriyakhetarpal ’s suggestion. Thank you!